lessons/Algebra/surface-area-rect-prism/langs/en-us/pages/sa-rubric.adoc

```diff
@@ -0,0 +1,45 @@
+[.landscape]
+= Rubric: Surface Area of a Rectangular Prism
+++++
+<style>
+	td { font-size: 9pt !important; }
+	td:first-of-type { font-weight: bold !important; }
+</style>
+++++
+
+Important: Students, be sure to sumbit all components of this project, including (1) your published Pyret program link, (2) the rectangular prism that you built, and (3) Surface Area of a Prism - Practice.
+
+
+@vspace{1ex}
+
+[cols="2,4,4,4", options="header"]
+|===
+|
+| Wow!
+| Getting There
+| Needs Improvement
+
+
+| My Program Submission
+| Running my code produces no errors. In Part 1, I correctly defined all six faces of a rectangular prism. In Part 2, I correctly coded the list of faces. In Part 3, I wrote definitions that calculate the area of each face. In Part 4, I wrote code that correctly computes surface area.
+| Running my code produces no errors. There is a small error in one or two of the sections of my Definitions Area.
```

Comment:
If it produces no errors, how can there be errors?

Reply:
Lol, I changed my mind. This rubric row should focus only on code, not conceptual understanding, which is asseessed in other rows. Gonna simplify this.